easybuild/easyconfigs/c/configparser/configparser-3.5.0-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'PythonPackage'
+
+name = 'configparser'
+version = '3.5.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://docs.python.org/3/library/configparser.html'
+description = """configparser is a Python library that brings the updated configparser from Python 3.5 to Python 2.6-3.5."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['https://pypi.python.org/packages/7c/69/c2ce7e91c89dc073eb1aa74c0621c3eefbffe8216b3f9af9d3885265c01c/']
+sources = ['configparser-3.5.0.tar.gz']
```

Comment:
@gatagat this hardcoded version is bound to bite us in the ass later, e.g. by only updating the `version = ...` line and not updating this

We can avoid this by using the `%(version)s` template, fixed with https://github.com/gatagat/easybuild-easyconfigs/pull/2, please review/merge